TradeItIosEmsApi/TradeItConnector.h

```diff
@@ -116,4 +116,16 @@
 - (void)sendEMSRequest:(NSMutableURLRequest * _Nullable)request
    withCompletionBlock:(void (^ _Nullable)(TradeItResult * _Nullable, NSMutableString * _Nullable))completionBlock;
 
+- (void)getOAuthLoginPopupUrlForMobileWithBroker:(NSString * _Nullable)broker
+                         interAppAddressCallback:(NSString * _Nullable)interAppAddressCallback
+                                 completionBlock:(void (^ _Nullable)(TradeItResult * _Nullable))completionBlock;
+
+- (void)getOAuthLoginPopupURLForTokenUpdateWithBroker:(NSString * _Nullable)broker
+                                               userId:(NSString * _Nullable)userId
+                              interAppAddressCallback:(NSString * _Nullable)interAppAddressCallback
+                                      completionBlock:(void (^ _Nullable)(TradeItResult * _Nullable))completionBlock;
+
+- (void)getOAuthAccessTokenWithOAuthVerifier:(NSString * _Nullable)oAuthVerifier
+                             completionBlock:(void (^ _Nullable)(TradeItResult * _Nullable))completionBlock;
+
```

Comment:
Sure, if you want to maintain more Obj-C code? There are a bunch of new classes in this PR that could have been Swift